app/controllers/sessions_controller.rb

```diff
@@ -6,6 +6,7 @@ def create
 
     if user&.authenticate params[:session][:password]
       log_in user
+      params[:session][:remember_me] == "1" ? remember(user) : forget(user)
```

Comment:
params[:session][:remember_me] tại sao lại cần session ở đằng trước

Reply:
Cần session vì khi ta tạo label ở sessions/new.html.erb ta định nghĩa form for :session, ô input có name="session[remember_me]", nếu params[:remember_me] thì sẽ không thể lấy được value checkbox => ko thể remember(user)